tests/annulus.cc

```diff
@@ -0,0 +1,562 @@
+#include <aspect/material_model/simple.h>
```

Comment:
if this is a copy of the file in benchmarks, please use a `#include ../benchmarks/.../...cc` instead. That way we can test the actual code in benchmarks.